src/alchemlyb/tests/parsing/test_gmx.py

```diff
@@ -3,118 +3,144 @@
 """
 
 import bz2
-import pytest
 
-from alchemlyb.parsing.gmx import extract_dHdl, extract_u_nk, extract
+import pytest
 from alchemtest.gmx import load_benzene
-from alchemtest.gmx import load_expanded_ensemble_case_1, load_expanded_ensemble_case_2, load_expanded_ensemble_case_3
-from alchemtest.gmx import load_water_particle_with_total_energy
+from alchemtest.gmx import (
+    load_expanded_ensemble_case_1,
+    load_expanded_ensemble_case_2,
+    load_expanded_ensemble_case_3,
+)
 from alchemtest.gmx import load_water_particle_with_potential_energy
+from alchemtest.gmx import load_water_particle_with_total_energy
 from alchemtest.gmx import load_water_particle_without_energy
 from numpy.testing import assert_almost_equal
 
+from alchemlyb.parsing.gmx import extract_dHdl, extract_u_nk, extract
+
 
 def test_dHdl():
-    """Test that dHdl has the correct form when extracted from files.
-
-    """
+    """Test that dHdl has the correct form when extracted from files."""
     dataset = load_benzene()
```

Comment:
This test file has a lot of instances of `dataset = load_something()` that could _eventually_ be turned into fixtures. It makes no big difference for performance, I think, but it would make our approach more uniform.
```python
@pytest.fixture
def gmx_benzene_data():
   return gmx.load_benzene()

def test_dHdl(gmx_benzene_data):
    ...
```

However, no need to do it in this PR, just for the future.

Reply:
So the logic is that the `conftest` will do and will only do the parsing, additional operations like `concat` will be done at the local level.